### ISSUE TITLE
Errors when starting the datacenter

### ISSUE THREAD

**Issue by ghost** [strands-project/strands_utils#27]
We get quite a lot of errors when we start the datacenter. I'm not sure if this is normal, but I thought we should report it, in case it isn't.

```
ERROR] [WallTime: 1383659550.111968] bad callback: <bound method DiagnosticsLogger._rosout_cb of <strands_diagnostics.logger.DiagnosticsLogger object at 0x1e63650>>
Traceback (most recent call last):
  File "/opt/ros/groovy/lib/python2.7/dist-packages/rospy/topics.py", line 683, in _invoke_callback
    cb(msg)
  File "/opt/strands/strands_catkin_ws/src/strands_utils/strands_diagnostics/src/strands_diagnostics/logger.py", line 60, in _rosout_cb
    store_message_no_meta(collection, msg)
  File "/opt/strands/strands_catkin_ws/src/strands_utils/strands_datacentre/src/strands_datacentre/util.py", line 75, in store_message_no_meta
    doc=yaml.load(str(msg))  # TODO: this is inefficient, should improve
  File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 71, in load
    return loader.get_single_data()
  File "/usr/lib/python2.7/dist-packages/yaml/constructor.py", line 37, in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/usr/lib/python2.7/dist-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/lib/python2.7/dist-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/usr/lib/python2.7/dist-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/usr/lib/python2.7/dist-packages/yaml/scanner.py", line 220, in fetch_more_tokens
    return self.fetch_value()
  File "/usr/lib/python2.7/dist-packages/yaml/scanner.py", line 580, in fetch_value
    self.get_mark())
ScannerError: mapping values are not allowed here
  in "<string>", line 9, column 20:
    msg: waitForService: Service [/head_xtion_nodelet_m ...
                       ^

[ERROR] [WallTime: 1383659550.119522] bad callback: <bound method DiagnosticsLogger._rosout_cb of <strands_diagnostics.logger.DiagnosticsLogger object at 0x1e63650>>
Traceback (most recent call last):
  File "/opt/ros/groovy/lib/python2.7/dist-packages/rospy/topics.py", line 683, in _invoke_callback
    cb(msg)
  File "/opt/strands/strands_catkin_ws/src/strands_utils/strands_diagnostics/src/strands_diagnostics/logger.py", line 60, in _rosout_cb
    store_message_no_meta(collection, msg)
  File "/opt/strands/strands_catkin_ws/src/strands_utils/strands_datacentre/src/strands_datacentre/util.py", line 75, in store_message_no_meta
    doc=yaml.load(str(msg))  # TODO: this is inefficient, should improve
  File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 71, in load
    return loader.get_single_data()
  File "/usr/lib/python2.7/dist-packages/yaml/constructor.py", line 37, in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/usr/lib/python2.7/dist-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/usr/lib/python2.7/dist-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/usr/lib/python2.7/dist-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/usr/lib/python2.7/dist-packages/yaml/scanner.py", line 220, in fetch_more_tokens
    return self.fetch_value()
  File "/usr/lib/python2.7/dist-packages/yaml/scanner.py", line 580, in fetch_value
    self.get_mark())
ScannerError: mapping values are not allowed here
  in "<string>", line 9, column 20:
    msg: waitForService: Service [/head_xtion_nodelet_m ...
```

Repeated several times.

**Comment by RaresAmbrus** [comment]
We get the same errors...

**Comment by Jailander** [comment]
I was having that error and temporarilyI commented the diagnostics_logger on the strands_datacentre launch file whilst I created an issue just like this one thank you

**Comment by hawesie** [comment]
Closed as duplicate of #23
